controlplane/src/db/schema.ts

```diff
@@ -464,8 +464,41 @@ export const organizationWebhooks = pgTable('organization_webhook_configs', {
   createdAt: timestamp('created_at', { withTimezone: true }).notNull().defaultNow(),
 });
 
+export const webhookGraphSchemaUpdate = pgTable(
+  'webhook_graph_schema_update',
+  {
+    webhookId: uuid('webhook_id')
+      .notNull()
+      .references(() => organizationWebhooks.id, {
+        onDelete: 'cascade',
+      }),
+    federatedGraphId: uuid('federated_graph_id')
+      .notNull()
+      .references(() => federatedGraphs.id, {
+        onDelete: 'cascade',
+      }),
+  },
+  (t) => {
+    return {
+      pk: primaryKey(t.webhookId, t.federatedGraphId),
+    };
+  },
+);
+
+export const webhookGraphSchemaUpdateRelations = relations(webhookGraphSchemaUpdate, ({ one }) => ({
+  organizationWebhook: one(organizationWebhooks, {
+    fields: [webhookGraphSchemaUpdate.webhookId],
+    references: [organizationWebhooks.id],
+  }),
+  federatedGraph: one(federatedGraphs, {
```

Comment:
We should design it in a way that all graphs can be referenced. This allows for notifications of fed and subgraphs. Use the target id.

Reply:
Why are we reusing a table for more than one event type? This is pre optimization. We need to keep things separate and clear to avoid confustion

Reply:
ok, your choice.